lib/core.js

```diff
@@ -431,15 +439,6 @@ declare class Promise<R> {
     static reject<T>(error?: any): Promise<T>;
     static all<T, Elem: Promise<T> | T>(promises: Array<Elem>): Promise<Array<T>>;
     static race<T, Elem: Promise<T> | T>(promises: Array<Elem>): Promise<T>;
-
-    // Non-standard APIs common in some libraries
```

Comment:
The reason I vote to remove is not that this declaration is incompatible //if it exists//, it's that it might not exist at all. `.done()` should error unless you're in an environment that polyfills `.done()`.